orchagent/bulker.h

```diff
@@ -104,6 +188,54 @@ namespace std
             return seed;
         }
     };
+
+    template <>
+    struct hash<sai_inbound_routing_entry_t>
+    {
+        size_t operator()(const sai_inbound_routing_entry_t& a) const noexcept
+        {
+            size_t seed = 0;
+            boost::hash_combine(seed, a.switch_id);
+            boost::hash_combine(seed, a.eni_id);
+            return seed;
+        }
+    };
+
+    template <>
+    struct hash<sai_outbound_ca_to_pa_entry_t>
+    {
+        size_t operator()(const sai_outbound_ca_to_pa_entry_t& a) const noexcept
+        {
+            size_t seed = 0;
+            boost::hash_combine(seed, a.switch_id);
+            boost::hash_combine(seed, a.dst_vnet_id);
+            return seed;
```

Comment:
Similar to above - behavioral model uses VNET ID and DIP as table keys, is it necessary to add DIP here?

Reply:
Fixed this.